### ISSUE TITLE
eva.perturb

### ISSUE THREAD

**Issue by junha-lee** [ZhuohanX/DeltaScore#2]
there is no eva.perturb directory

how can i get it

**Comment by Rollingpig** [comment]
yes I have the same concern. I checked the arxiv but there is no way to reconstruct the missing file

**Comment by ZhuohanX** [comment]
Thanks for pointing this out. `eva.perturb` was an auxiliary perturbation utility used in the original experiments, and it was accidentally omitted from the public repository.

I found an old backup and opened #3 to restore the recoverable source files and small resource tables. The large commonsense auxiliary resources (`conceptnet_triple.csv`, `kg.txt`, and `word2kg.txt`) are not included in the PR because they are about 300 MB in the archived experiment directory, and the old external mirror links no longer reliably return raw data files. So this should recover several perturbation paths, but exact full reproduction may still require the original dependency/data setup.
